fpga.chart.py

```diff
@@ -11,40 +11,45 @@
 import copy
 import subprocess
 import re
+import os
 
 priority = 90000
 
 FPGA_COUNT = 2
```

Comment:
Remove

Reply:
This is used to check that the command exists.